### PR TITLE
added esConsumes to modules in IMOC/EventVertexGenerators

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 2 modules
#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
This change was made under my supervision.

File: IOMC/EventVertexGenerators/src/BetafuncEvtVtxGenerator.cc [hunk 1]
```diff
@@ -50,19 +47,20 @@ BetafuncEvtVtxGenerator::BetafuncEvtVtxGenerator(const edm::ParameterSet& p) : B
...
+    beamToken_ = esConsumes<SimBeamSpotObjects, SimBeamSpotObjectsRcd, edm::Transition::BeginLuminosityBlock>();
+  }
 }
 
 BetafuncEvtVtxGenerator::~BetafuncEvtVtxGenerator() {}
 
 void BetafuncEvtVtxGenerator::beginLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const& iEventSetup) {
   update(iEventSetup);
 }
-void BetafuncEvtVtxGenerator::beginRun(const edm::Run&, const edm::EventSetup& iEventSetup) { update(iEventSetup); }
```
Comment (Dr15Jones) [inline IOMC/EventVertexGenerators/src/BetafuncEvtVtxGenerator.cc:60]: Getting from the EventSetup in both the BeginRun and BeginLuminosityBlock is unnecessary and would have required 2 different tokens.

**Comment by Dr15Jones** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**